components/Heroes/BenchmarkBadge.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Grid, Row, Col } from 'react-flexbox-grid';
```

Comment:
I think we should either use it everywhere or nowhere.  If we can achieve the same effect without adding more dependencies then we shouldn't use it